db/migrate/20160801150013_add_validations_to_financial_transactions.rb

```diff
@@ -1,6 +1,8 @@
 class AddValidationsToFinancialTransactions < ActiveRecord::Migration
   def change
-    change_column(:financial_transactions, :amount, :integer, null: false, default: 0)
-    change_column(:financial_transactions, :adjustment, :integer, null: false, default: 0)
+    change_column_null(:financial_transactions, :amount, false, 0)
+    change_column_null(:financial_transactions, :adjustment, false, 0)
+    change_column(:financial_transactions, :amount, :integer, default: 0)
+    change_column(:financial_transactions, :adjustment, :integer, default: 0)
```

Comment:
Works as promised! Unfortunately, anyone who has pulled down master will still have reset, but I think the exercise was worth it. If someone still has old `financial_transactions` (old to your previous PR) in their db, this migration will not cause an issue for them. Confirmed!